### ISSUE TITLE
readme bash function error

### ISSUE THREAD

**Issue by vgeck** [jonathf/envmgr#1]
Hei @jonathf 
do not know if you still use this pkg .. I do :)

And:
the bash alias function in readme should be changed to:

```
function enw() {
    envwrap "$@"
    autoenv_init
}
```

it is `envwrapper` to `envwrap`

**Comment by jonathf** [comment]
Right. Cool. Not for a while no, but now I am considering going back for `chaospy` and `numpoly` again. The multi-python-version support is still superior to what is out there, and this thing should work great in tandem with poetry.

I've pushed the change to master now.
